src/main/java/umaikaze/duke/Parser.java

```diff
@@ -0,0 +1,73 @@
+package umaikaze.duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
```

Comment:
The `Parser` class is well-organized. But it is still suggested to add in more detailed `javadoc` so peers or readers know what does ` a parser` convert from and to.

Reply:
To add on, sometime `parser` is considered to convert between `String` and `Task` objects, so it is better to make it clearer. Maybe you can change it to `formatParser` ?